### DISCUSSION TITLE
CPU usage when running in another server

### DISCUSSION THREAD

**Discussion by Minglarn** [blakeblackshear/frigate#775] [General]
Hi! 
I decided to move my hole Frigate docker to a ubuntu server. An old laptop running ubuntu 20 headless as it got a broken screen. 
Works flawless and I've managed to reduce the load on my NUC with over 60%.

I know that my laptop is sometimes running on heavy load when frigate decodes 9 2560x1920 streams but for the most part is laying around 30-60%. 

So the question is: 
Is there any possibility to monitor CPU usage? Can Frigate create a sensor with CPU usage from docker?

**Comment by ekos2001** [comment]
You can install glances to your ubuntu server and use this integration https://www.home-assistant.io/integrations/glances

**Reply by Minglarn** [reply]
Yep, that one way to do it... But it would be considerably easier if Frigate included this info as a sensor. 
This could even be a good way to track the CPU usage for the docker/server to see if the machine is under heavy load (CPU) .
